openmetadata-docs/content/v1.1.0/deployment/docker/index.md

```diff
@@ -74,35 +77,88 @@ Follow the instructions [here](https://docs.docker.com/compose/cli-command/#inst
     > Docker Compose version v2.2.3
     ```
 
+## Minimum Sizing Requirements
+
+We recommend you to allocate openmetadata-server with minimum of 2vCPUs and 6 GiB Memory.
+
+For External Services that openmetadata depends on -
+- For the database, minimum 2 vCPUs and 2 GiB RAM (per instance) with 30 GiB of Storage Volume Attached (dynamic expansion up to 100 GiB)
+- For Elasticsearch, minimum 2 vCPUs and 2 GiB RAM (per instance) with 30 GiB of Storage volume attached
+
 ## Steps for Deploying OpenMetadata using Docker 
 
-- First download the docker-compose.yml file from the release page [here](https://github.com/open-metadata/OpenMetadata/releases/latest). The latest version is at the top of the page
-  - Deploying with MySQL:  Download `docker-compose.yml` file from the above link.
-  - Deploying with PostgreSQL: Download `docker-compose-postgres.yml` file from the above link.
+### 1. Create a directory for OpenMetadata
+
+Create a new directory for OpenMetadata and navigate into that directory.
 
-- Create the directory for host volumes 
 ```commandline
-mkdir -p $PWD/docker-volume/db-data
+mkdir openmetadata-docker && cd openmetadata-docker
 ```
 
-- Run the below command to deploy the OpenMetadata
+### 2. Download Docker Compose File from Github Release Branch
```

Comment:
we shouldn't be asking users download it from release branch. instead ask them to go to releases and download the file. 
1. We should add docker compose that contains OM Server
2. We should add docker compose that contains Ingestion

DB & Search dependencies users needs to provide

Reply:
So Apart from the 2 docker compose files we add as part of Release assets, we need to add 4 more ?

- 2 for Docker Compose that contains OpenMetadata Server (mysql, postgresql)
- 2 for Docker Compose that container Ingestion Service (mysql,postgresql)

Or just tar the directories `docker/docker-compose-openmetadata` and `docker/docker-compose-ingestion` and add it as part of assets ?